components/Dialogs/src/MDCAlertController.h

```diff
@@ -198,6 +200,9 @@
  Descriptive attributed text that summarizes a decision in a sentence of two.
 
  If provided and non-empty, will be used instead of @c message property.
+ 
+ @note Currently, support for tappable embedded links within @c attributedMessage
```

Comment:
Better:

" @note Currently tappable embedded links within @c attributedMessage are not supported."

Reply:
Will do!